distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -6,130 +6,149 @@
 :order: 01
 
 Introduction: ${catalog-ui} represents the most advanced search interface available with ${branding}.
-It provides a metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
+It provides metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
 
 [NOTE]
 ====
-For more detail on any feature or button within the Catalog UI, click the `?` button in the upper right of the screen; then, hover over any item on the screen and a contextual tooltip will to define its purpose.
+For more detail on any feature or button within ${catalog-ui}, click the `?` icon in the upper right of the screen; then, hover over any item on the screen and a contextual tooltip will be displayed to define its purpose.
 To exit this mode, click the `?` again or press `escape`.
 ====
 
 === Accessing ${catalog-ui}
 
-The default location for ${catalog-ui} is ${secure_url}/search/catalog
+The default URL for ${catalog-ui} is ${secure_url}/search/catalog
 
 .Catalog UI Guest Users
 [NOTE]
 ====
-If Guest access has been enabled, users not signed in to ${branding} (guest users) will have access to search functions, but all workspace configuration and settings will only exist locally and not be available for sharing.
+If Guest access has been enabled, users not signed in to ${branding} (ie. guest users) will have access to search functions, but all workspace configuration and settings will only exist locally and will not be available for sharing.
```

Comment:
❓ should this be `i.e.` I need to work on my grammar

Reply:
yes, fixed